overlay/src/renderer/styles.css

```
@@ -64,17 +147,86 @@
+	animation: success-ring 0.42s ease-out forwards;
+}
+
+.status-indicator.success .checkmark-circle-icon {
+	position: relative;
+	color: #10b981;
+	filter: drop-shadow(0 0 6px rgba(16, 185, 129, 0.45));
+}
+
+.status-indicator.success .checkmark-circle-icon circle,
+.status-indicator.success .checkmark-circle-icon polyline {
+	stroke: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)